### ISSUE TITLE
Jupyter Website Revamp

### ISSUE THREAD

**Issue by cameronoelsen** [jupyter/design#16]
Here is where all of the progress on the new Jupyter site will be held! There are now two separate threads for IPy and Jupyter.

--- 
[edit by Matthias] 
![jupyter home1](https://cloud.githubusercontent.com/assets/6437976/8343140/0fbf77e0-1a87-11e5-8357-a5ee4b1d3663.png)
https://jupyter.hackpad.com/Jupyter-Content-and-Organization-Gg50rKW9YW7
[/edit]

**Comment by Carreau** [comment]
In response to @fperez : 

The hovering icons-coloring might be painful to do if it is not a square map and in particular difficult if we need to update. Also why some languages and not others ? I would prefer a page like @bollwyvl's one with a design that matches.

**Comment by fperez** [comment]
Sure, if it's too painful to do, it was just an idea... More important that we get the basic thing in place.  Would be nice if at least they were links, though...

**Comment by cameronoelsen** [comment]
Currently this is where I am in regards to the website design of the front page!

![jupyter website mockup 1](https://cloud.githubusercontent.com/assets/6437976/8757404/d2955cc6-2c8e-11e5-98bd-54d0bc79d05c.png)

**Comment by rgbkrk** [comment]
Wow.

:open_mouth:

**Comment by Carreau** [comment]
OMG !

:cake:

**Comment by jhamrick** [comment]
This looks *awesome* :tada: 

What is the JupyterHub thing at the bottom supposed to be for, though? A link to JupyterHub docs? Currently there's no context around it so it seems a bit abrupt.

**Comment by minrk** [comment]
That's really nice! We may have to be careful with logos, though. I'm not sure we have permission to use all of those. Anyone more official know how to find that stuff out?

**Comment by ellisonbg** [comment]
We are tracking down the logo permissions privately with all of those
folks...a pain but getting there...

On Fri, Jul 17, 2015 at 3:49 PM, Min RK <notifications@github.com> wrote:

> That's really nice! We may have to be careful with logos, though. I'm not
> sure we have permission to use all of those. Anyone more official know how
> to find that stuff out?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122438783>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by bollwyvl** [comment]
Really great stuff!

Have you thought about how we might make this data driven? If we collect
all the data on kernels i suggest on that other pr, we can use similar
techniques to make those home page visuals interactive and easier to keep
up to date. The wee kernel icons already look like they might be force
directable... Requiring redesign every time we want to add a kernel logo
send like lots of effort.

Related, what are we envisioning as the tool chain for this new site? Seems
like getting down to where the content can be maintained in
markdown/templates, and built automatically (Travis) is a Good Thing... I'd
favor a node-based flow, rather than jekyll, but there is something to be
said for working with what the hosting provides for free.

**Comment by ellisonbg** [comment]
>
>
> Have you thought about how we might make this data driven? If we collect
> all the data on kernels i suggest on that other pr, we can use similar
> techniques to make those home page visuals interactive and easier to keep
> up to date. The wee kernel icons already look like they might be force
> directable... Requiring redesign every time we want to add a kernel logo
> send like lots of effort.
>

Yep, we already are:

* We have been using the Google Analytics from our existing websites to
help us organize the content.
* We plan on doing usability testing (quatitative and qualitative) on the
new website to continue to iterate and improve it.

Right now the kernel logos are placed by hand. If us coders were to do
that, yes it is a pain. However, seeing how fast our designer works makes
me less concerned about that.


>
> Related, what are we envisioning as the tool chain for this new site? Seems
> like getting down to where the content can be maintained in
> markdown/templates, and built automatically (Travis) is a Good Thing... I'd
> favor a node-based flow, rather than jekyll, but there is something to be
> said for working with what the hosting provides for free.
>

Right now the website is just hand coded HTML and CSS with no templates.
Here are my current thoughts on tool chain and hosting:

* Use GH pages.
* Keep it to hand coded HTML/CSS, but add templates
* Build using travis.
* I am fine with a node based build chain.

Nicholas, once we get a bit further, would you be willing to help get the
toolchain/Travis setup? Our designer (who is doing the HTML/CSS) doesn't
have any coding experience so it would be super helpful.




>
>
>
>
> On 18:56, Fri, Jul 17, 2015 Brian E. Granger <notifications@github.com>
> wrote:
>
> > We are tracking down the logo permissions privately with all of those
> > folks...a pain but getting there...
> >
> > On Fri, Jul 17, 2015 at 3:49 PM, Min RK <notifications@github.com>
> wrote:
> >
> > > That's really nice! We may have to be careful with logos, though. I'm
> not
> > > sure we have permission to use all of those. Anyone more official know
> > how
> > > to find that stuff out?
> > >
> > > —
> > > Reply to this email directly or view it on GitHub
> > > <https://github.com/jupyter/design/issues/16#issuecomment-122438783>.
> > >
> >
> >
> >
> > --
> > Brian E. Granger
> > Cal Poly State University, San Luis Obispo
> > @ellisonbg on Twitter and GitHub
> > bgranger@calpoly.edu and ellisonbg@gmail.com
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/jupyter/design/issues/16#issuecomment-122439730>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122557859>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by damianavila** [comment]
Really, really nice... great work

> Here are my current thoughts on tool chain and hosting:

I agree with these steps...

**Comment by cameronoelsen** [comment]
![iteration 2](https://cloud.githubusercontent.com/assets/6437976/8763936/a8d05058-2d6a-11e5-83d3-ec31df176b0f.png)
Here is more of the front page from today's work I did, I am going to continue to iterate and improve upon it!

**Comment by Carreau** [comment]
My brain tells me the text in `jupyterhub` logo is off centered and too high, maybe an effect of the "ring" ?

**Comment by minrk** [comment]
Yeah, and "Enterprise" is definitely not an appropriate word to use for JupyterHub. "Hosted", perhaps.

**Comment by fperez** [comment]
I love where this is going, but I'm with @minrk on preferring "hosted" to "enterprise"

**Comment by ellisonbg** [comment]
I realize that "enterprise" is a stretch, but I don't think that "hosted"
is the right word as it implied that we are hosting it for people. One of
the points below says "centralized deployment" so I don't want to repeat
that. I am looking for a word that expresses that JupyterHub us useful to

groups of people -> organizations -> enterprise

Maybe "Jupyter Notebooks for Your Organization"?

On Sat, Jul 18, 2015 at 8:14 PM, Fernando Perez <notifications@github.com>
wrote:

> I love where this is going, but I'm with @minrk <https://github.com/minrk>
> on preferring "hosted" to "enterprise"
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122618572>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by ellisonbg** [comment]
Also, remember, our website can bend reality to our liking in subtle ways
;-)

Software gains buzzwords attributes not by technical merit or features, but
rather by simple assertion of the marketing department. A reminder of how
it works:

http://www.mongodb-is-web-scale.com/

I hate this as much as anyone, but some of our potential "customers" won't
touch stuff that is not "enterprise grade." Because enterprise grade is
mostly a meaningless, all we have to do is say it is so...

Which is why I wish we could give all our press releases as Onion Articles:
"Open-source Project Jupyter declares it is enterprise grade, just cause"

On Sat, Jul 18, 2015 at 8:25 PM, Brian Granger <ellisonbg@gmail.com> wrote:

> I realize that "enterprise" is a stretch, but I don't think that "hosted"
> is the right word as it implied that we are hosting it for people. One of
> the points below says "centralized deployment" so I don't want to repeat
> that. I am looking for a word that expresses that JupyterHub us useful to
>
> groups of people -> organizations -> enterprise
>
> Maybe "Jupyter Notebooks for Your Organization"?
>
> On Sat, Jul 18, 2015 at 8:14 PM, Fernando Perez <notifications@github.com>
> wrote:
>
>> I love where this is going, but I'm with @minrk
>> <https://github.com/minrk> on preferring "hosted" to "enterprise"
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/jupyter/design/issues/16#issuecomment-122618572>.
>>
>
>
>
> --
> Brian E. Granger
> Cal Poly State University, San Luis Obispo
> @ellisonbg on Twitter and GitHub
> bgranger@calpoly.edu and ellisonbg@gmail.com
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
Then that's it, use "Large scale Jupyter Notebook deployment". Also, feel free to push toward more Jupyter-ish term. Heroku have Dynos, BuildPack, Manifold, Jupyter Could have Accretion Disk, Constelation, Chromosphere...

I can perfectly see "Deploy you Jupyter Notebooks Constellation". Renaming JupyterHub to Jupyter-Perseyds might be too much though :-P

**Comment by fperez** [comment]
Good point, while we could try to clarify that we mean "hosted by yourself", rather than "hosted by us", I can see the chance for confusion...

Just to throw a few names out (some of these are bad, I know, just trying to get us thinking about options):

- "Jupyter Notebooks X", X being: 'for groups', 'for your organization', 'you can host', 'inside the firewall', ...
- "X Jupyter Notebooks", X being: "group", "multiuser", "in-house",

**Comment by ellisonbg** [comment]
I would like to find something that would resonate with decision makers and
devops/IT in organizations who have heard that their employees are using
the notebook and they come to our website to find out "what is this jupyter
thing and why would we want to use it." The only name that captures the
assumptions of those folks is the "for your organization" one...or the
"enterprise" one...

Part of this comes from our looking at the page flow of the Google
Analytics of the existing ipython.org. Two of the most popular routes are
Google -> notebook.html and index.html -> notebook.html. This shows that a
ton of people get to our website asking the question "what is the notebook?"

On Sat, Jul 18, 2015 at 9:47 PM, Fernando Perez <notifications@github.com>
wrote:

> Good point, while we could try to clarify that we mean "hosted by
> yourself", rather than "hosted by us", I can see the chance for confusion...
>
> Just to throw a few names out (some of these are bad, I know, just trying
> to get us thinking about options):
>
>    - "Jupyter Notebooks X", X being: 'for groups', 'for your
>    organization', 'you can host', 'inside the firewall', ...
>    - "X Jupyter Notebooks", X being: "group", "multiuser", "in-house",
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122625487>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by minrk** [comment]
The main reason I don't think enterprise is appropriate is that the primary information "Enterprise" communicates to me is commercial institutional support, which we don't offer.

**Comment by Carreau** [comment]
Agreed. "Entreprise" also push away institutions like University etc...

**Comment by rgbkrk** [comment]
If you want a fancy word that applies well, I'd go with "Multi-tenant Jupyter Notebooks" or "Multi-user Jupyter Notebooks" (though configuration is up to them to adhere to either of those).

:-1: on enterprise

Tenancy will matter to IT and hosting professionals. Rub some Docker on it too.

**Comment by damianavila** [comment]
:-1: to enterprise as well, "Multi-tenant" is interesting...

**Comment by ellisonbg** [comment]
I think my favorite for the main heading is probably "Jupyter Notebooks for
your Organization"

multi-user = not quite formal enough
multi-tenant = umm, I don't think JupyterHub is multi-tenant be default, so
I am puzzled by this one

On Sun, Jul 19, 2015 at 3:35 PM, Damián Avila <notifications@github.com>
wrote:

> [image: :-1:] to enterprise as well, "Multi-tenant" is interesting...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122711833>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
We know that Entreprise is your Duck Brian.

**Comment by ellisonbg** [comment]
No, it is more like this (I am being mostly serious):

We keep encountering people and big companies who say things like this:

"Wow, Jupyter is amazing, but is it Enterprise ready?"

When I heard it from 1 company it was easy to dismiss, but we are now
hearing from dozens of them, so it is hard to ignore...

On Sun, Jul 19, 2015 at 6:09 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> We know that Entreprise is your Duck Brian.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122724011>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
Responding to wether it is enterprise ready do not mean we have to put Enterprise in the title.
I would in particular expect "Entreprise" ready to be a much simpler to deploy that what we have now. 

"Entreprise Frobulator" often mean you pay a license, get a docker image and run it. Which is not what JupyterHub does. Having Notebook deployment, Web Scale, Or in the cloud. or for your Organisation covey a better message.

**Comment by ellisonbg** [comment]
Agreed :)

On Sun, Jul 19, 2015 at 7:26 PM, Matthias Bussonnier <
notifications@github.com> wrote:

> Responding to wether it is enterprise ready do not mean we have to put
> Enterprise in the title.
> I would in particular expect "Entreprise" ready to be a much simpler to
> deploy that what we have now.
>
> "Entreprise Frobulator" often mean you pay a license, get a docker image
> and run it. Which is not what JupyterHub does. Having Notebook deployment,
> Web Scale, Or in the cloud. or for your Organisation covey a better
> message.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-122731803>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by takluyver** [comment]
I think everyone knows that the [Enterprise](https://en.wikipedia.org/wiki/USS_Enterprise_%28NCC-1701%29) is far beyond Jupyter ;-)

**Comment by rgbkrk** [comment]
:+1: to @takluyver's inclusion of the one true enterprise

**Comment by cameronoelsen** [comment]
You can track my progress on the development of the new website through this github page:
http://cameronoelsen.github.io/jupytersite/

**Comment by damianavila** [comment]
You have a typo mistake: "Jupyter noteboo" under the kernels logos (left)... I can paste a pic, but you can easily find it... let me know if not...

**Comment by cameronoelsen** [comment]
@damianavila Just overhauled that header, hopefully fixed your problem!

**Comment by bollwyvl** [comment]
@cameronoelsen thanks for the preview site! It's really looking nice.

I'm seeing horizontal scrollbars at all resolutions: adding a `<div class="container-fluid">` wrapping everything in the body cleans it up.

Also, I think we are somewhat remiss in omitting both "Open Source Software" and/or "Free Software" from the home page... i would pick one of those, and find a place to put it in the first blurb. And or make it a header link that be on every page.

To that end, putting a Creative Commons dedication on the page content would help set the stage: perhaps [CC4](http://creativecommons.org/licenses/by/4.0/)?

**Comment by bollwyvl** [comment]
Oh, and the typekit thing doesn't appear to be working. It seems like using a non-free font is bad for business, anyway. What font are you trying to use? I'm sure we can find a free altenative!

**Comment by ellisonbg** [comment]
Unfortunately the selection of free fonts really lags behind the non-free ones. That isn't to say that there aren't some good free fonts - just too few. I am -1 on constraining our designers to only using free tools and fonts. 

Plus typekit, sketch, creative suite, etc are cheap enough that we can purchase them for anyone, including community members, that want to work on design.

However a few people have brought this up recently so we may want to discuss this on the main jupyter google group.

We may need to update the type kit configure to work on this temporary domain.

**Comment by rgbkrk** [comment]
I'd like to think that I'm a strong ideologue within open source, preferring free and open source software wherever possible. In this specific case though, the decision we made about picking a single font was so that we could establish a brand for logos. The tradeoff is marketability (many free fonts didn't match the style we were aiming for).

**Comment by bollwyvl** [comment]
Is there a way to not have to host it externally? Typekit is blocked by my
default battery of things as it basically give adobe a tracking data point
where the use of the data is unclear.

**Comment by Carreau** [comment]
Can we rediscuss the use of Typekit ? It seem like highly more restrictive that what was advertised. 
I'm fine actually **buying** a font if you want a non open-source font, but the subscription seem to be annoying everyone but you.

**Comment by ellisonbg** [comment]
Sure, because this decision spans multiple repos/areas of the project, can
you start a discussion on the main jupyter google group?







On Fri, Aug 7, 2015 at 9:50 AM, Matthias Bussonnier <
notifications@github.com> wrote:

> Can we rediscuss the use of Typekit ? It seem like highly more restrictive
> that what was advertised.
> I'm fine actually *buying* a font if you want a non open-source font, but
> the subscription seem to be annoying everyone but you.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-128762561>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
> On Aug 7, 2015, at 10:03, Brian E. Granger <notifications@github.com> wrote:
> 
> Sure, because this decision spans multiple repos/areas of the project, can
> you start a discussion on the main jupyter google group?

Yes I can, and will. 
-- 
M

**Comment by ellisonbg** [comment]
@bollwyvl can you explain more about your comment about tracking? Here is
the privacy policy for typekit service:

http://www.adobe.com/privacy/typekit.html

TL;DR = no cookies, nothing that tracks users, basic stats on font loading
times, number of times a site serves a font.

Compared to many of the website we use everyday, this seems quite
reasonable. It is also not significantly different from the information
that Google would be gathering when you use their free webfonts.

On Thu, Aug 6, 2015 at 1:59 PM, bollwyvl <notifications@github.com> wrote:

> Is there a way to not have to host it externally? Typekit is blocked by my
> default battery of things as it basically give adobe a tracking data point
> where the use of the data is unclear.
>
> On 14:42, Thu, Aug 6, 2015 Kyle Kelley <notifications@github.com> wrote:
>
> > I'd like to think that I'm a strong ideologue within open source,
> > preferring free and open source software wherever possible. In this
> > specific case though, the decision we made about picking a single font
> was
> > so that we could establish a brand for logos. The tradeoff is
> marketability
> > (many free fonts didn't match the style we were aiming for).
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/jupyter/design/issues/16#issuecomment-128470784>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-128505952>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
I think the question is not wether or not typekit tracks or not, but that by default some configuration will block it.

You can try to convince user as much as you want that Stackoverflow is a useful resource, soem companies block the domain, and there is no way to convince IT. Take the example of Sylvain and Jason that do not have access to Google Hangout.

**Comment by ellisonbg** [comment]
I understand that some corporate environments block some websites. It is
just as likely that those environments would block sites that host free
webfonts such as Google fonts. Has anyone observed such a blockage in the
wild though?

CSS font specifications should always include reasonable backup fonts, even
if we use only free webfonts.

**Comment by bollwyvl** [comment]
So I've pulled the `jupytersite` repo and found that the typekit fonts aren't even being used. If the font is just used in embedded image text, is including the typekit thing some kind of licensing requirement?

**Comment by ellisonbg** [comment]
The Myriad Pro in the logo has been converted to raw paths in our SVG logo,
so that actual font is no longer needed for the logo (unless we want to
modify it). As far as I know, this was done on a system where we had the
proper license to do this.

The only reason we used to use TypeKit for the Jupyter website was for
other fonts on the site (Minion Pro and Proxima Nova). I am not sure the
current site is using either of these.

On Sat, Aug 8, 2015 at 5:29 PM, bollwyvl <notifications@github.com> wrote:

> So I've pulled the jupytersite repo and found that the typekit fonts
> aren't even being used. If the font is just used in embedded image text, is
> including the typekit thing some kind of licensing requirement?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-129078342>.
>



-- 
Brian E. Granger
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by minrk** [comment]
@cameronoelsen the page is looking nice. I see it still has the footer links from the current page. I don't think they quite fit in with the new style. Perhaps they should be a bit more prominent in a a 'follow Jupyter' or 'Community' block?

**Comment by ellisonbg** [comment]
@cameronoelsen how is everything going? Were you able to make any further
progress with @jdfreder there this week?

On Tue, Aug 11, 2015 at 1:36 PM, Min RK <notifications@github.com> wrote:

> @cameronoelsen <https://github.com/cameronoelsen> the page is looking
> nice. I see it still has the footer links from the current page. I don't
> think they quite fit in with the new style. Perhaps they should be a bit
> more prominent in a a 'follow Jupyter' or 'Community' block?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-130059606>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by rgbkrk** [comment]
It's nearing a point now where I'm wondering if we can push out this version and refine more after.

**Comment by ellisonbg** [comment]
Yes, I think we are getting very close to that point. We need to double
check the company logos to make sure we don't use any that we don't have
permissions to use yet. We started doing the design work of the company
logos before we had permissions to use them. By now, I think we have
permissions to use most, but there are still a few that have been finished
up. We should be able to do this in the next week - will be helped by my
return to SLO...

On Sat, Aug 15, 2015 at 7:29 AM, Kyle Kelley <notifications@github.com>
wrote:

> It's nearing a point now where I'm wondering if we can push out this
> version and refine more after.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-131387505>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by rgbkrk** [comment]
You have permission to use the Rackspace logo in reference to users of the notebook:

![image](https://cloud.githubusercontent.com/assets/836375/9552801/60728c64-4d80-11e5-9faf-18af05af34a1.png)

We're also missing [Time Magazine's Person of the Year for 2006](https://en.wikipedia.org/wiki/You_(Time_Person_of_the_Year)).

**Comment by ellisonbg** [comment]
Great, thanks for the logo+permission!

On Fri, Aug 28, 2015 at 10:30 AM, Kyle Kelley <notifications@github.com>
wrote:

> You have permission to use the Rackspace logo in reference to users of the
> notebook:
>
> [image: image]
> <https://cloud.githubusercontent.com/assets/836375/9552801/60728c64-4d80-11e5-9faf-18af05af34a1.png>
>
> We're also missing Time Magazine's Person of the Year for 2006
> <https://en.wikipedia.org/wiki/You_(Time_Person_of_the_Year)>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-135839838>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by damianavila** [comment]
OK, I will post the new Continuum logo here as well...

![continuum_logo_0702](https://cloud.githubusercontent.com/assets/1640669/9554081/baedb934-4d99-11e5-9fb6-a71e8a7f832d.png)

If you need ai or eps versions, let me know...

**Comment by Carreau** [comment]
> We're also missing Time Magazine's Person of the Year for 2006.

Canvas + webcam + BnW filter ?

**Comment by rgbkrk** [comment]
omg @Carreau, that's beautiful. Click on "You" and it adds them to a little canvas.

**Comment by bollwyvl** [comment]
Maybe a bit overkill for the home page, but a webcam widget notebook
example on tmnb would be very cool... Been thinking about webcam AR for
posters... A few subtle AR tags and qr codes on static, hi dpi printing
could trigger widgets that shared axes with the physical world.

**Comment by cameronoelsen** [comment]
Thanks for the logos @damianavila and @rgbkrk, Ill make sure to include those logos!

**Comment by jdfreder** [comment]
@cameronoelsen I managed to graft your commits on top of our current repo - see https://github.com/jupyter/jupyter.github.io/pull/15 .  Before you continue to alter anything, you'll need to talk to @ellisonbg to make sure that you are making modifications to that repo, and not the random one on your machine.  As you recall Thursday evening, the grafting procedure wasn't really straight forward. :skull:

**Comment by cameronoelsen** [comment]
After speaking with Brian and hearing that we should highlight our community of contributors along with the steering council that leads the project, I have added two sections to the about page and also tweaked the design to be more consistent. The community section spiderweb is going to utilize d3js and show how everyone has contributed and helped one another. Hope you like the mockup! (All of the supporting text is filler that I made up, so don't worry about that :) )

![concept](https://cloud.githubusercontent.com/assets/6437976/9672531/729b43a2-5250-11e5-9d12-cf2593d3e31c.png)

**Comment by fperez** [comment]
We were discussing that it might be better to have a simpler list of just names and affiliations for the Steering Council.

I also don't like the "ultimately make the final decisions" language: the role of the Steering Council is to help when certain decisions aren't easily reached by the community, not to make decisions instead of the community.  That language is not really how we should describe the council at all.

I would suggest instead something like "The Steering Council provides guidance and decision making support to the open Jupyter community", or something along those lines.  Open to more ideas on better wording.

**Comment by cameronoelsen** [comment]
@fperez All of the text on the page I made up on the spot just for the sake of being filler, no intention on using any of it, and sounds good!

**Comment by Carreau** [comment]
> Also, Mattias looks great in his closeup in the steering council section

Indeed.

![1686767](https://cloud.githubusercontent.com/assets/335567/9679024/1848e260-52eb-11e5-90f8-f9edaf5cef5f.jpg)

**Comment by rgbkrk** [comment]
Maybe you just need a real picture @Carreau. :wink:

**Comment by ellisonbg** [comment]
Cameron is working on a different visual design for the Steering Council
that is much more subtle and downplayed.

Also, if the d3 network stuff is too CPU intensive, we can replace it by a
static screenshot and have a "View Interactive Network" button that loads
the d3 stuff optionally. The d3 viz that Auberon has done is super cool
though - you can see the Steering Council in the network itself.

On Fri, Sep 4, 2015 at 6:43 AM, Kyle Kelley <notifications@github.com>
wrote:

> Maybe you just need a real picture @Carreau <https://github.com/Carreau>. [image:
> :wink:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/design/issues/16#issuecomment-137741368>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by Carreau** [comment]
> Cameron is working on a different visual design for the Steering Council
> that is much more subtle and downplayed.

Ok, great ! 


> Also, if the d3 network stuff is too CPU intensive, we can replace it by a
> static screenshot and have a "View Interactive Network" button that loads
> the d3 stuff optionally. The d3 viz that Auberon has done is super cool
> though - you can see the Steering Council in the network itself.

Depending on the network viz, the CPU load can be only for a short time, if the temperature of the network arrangement simulation cool down fast enough. We can also pause the rendering on 
window focus lost. 

So I'm ok to try to have a dynamic graph, I just don't want to spend too much effort if it drains battery :-)

**Comment by bollwyvl** [comment]
One option would be to pre-render the SVG and embed it straight in the
html, then activate any d3 stuff on explicit user interaction. You lose
some of the benefits of having "pure" data-driven documents, but it is
certainly less CPU up front.
